ports/quake.2/quake2/quake2.gptk

```diff
@@ -1,24 +1,28 @@
-back = delete
-start = esc
-a = \"
-b = \"
-x = \"
-y = \"
-l1 = \"
-l2 = \"
-l3 = \"
-r1 = \"
-r2 = \"
-r3 = \"
-up = \"
-down = \" 
-left = \"
-right = \"
-left_analog_up = \"
-left_analog_down = \"
-left_analog_left = \"
-left_analog_right = \"
-right_analog_up = \"
-right_analog_down = \" 
-right_analog_left = \"
-right_analog_right = \"
+back = tab
+start = f1
+a = c
+b = space
+x = quote
+y = enter
+l1 = mouse_right
+l2 = =
+l3 = g
+r1 = mouse_left
+r2 = /
+r3 = h
+guide = esc
```

Comment:
Most devices don't have a guide button.
So relying on putting guide as menu button is bad practice and will fail on most devices.
Either implement a hotkey approach for back + hotkey or don't use the guide button at all.

Many devices also use the "guide" button as R3 for example when they don't have real analog sticks.